.github/workflows/r_artifacts.yml

```diff
@@ -12,6 +12,11 @@ jobs:
     runs-on: ubuntu-latest
     container: rocker/r-base
     steps:
+      - name: Install Git before checkout
+        shell: bash
+        run: |
+          apt-get update
+          apt-get install --no-install-recommends -y git
```

Comment:
/gha build r-artifacts

Reply:
that comment created this build: https://github.com/microsoft/LightGBM/actions/runs/376731177

Reply:
✔️ this succeeded

![image](https://user-images.githubusercontent.com/7608904/99891272-f3478d00-2c5f-11eb-91b4-cd6d0a6d6ede.png)